examples/kubernetes/otel-collector.yaml

```diff
@@ -0,0 +1,153 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: otel-collector-config
+data:
+  config.yaml: |-
+    receivers:
+      filelog:
+        include:
+          # - /var/log/pods/namespace_pod-name_pod-uid/container-name/*.log
```

Comment:
Unless you want to demonstrate something by these commented lines I think it is best to delete them. If it shows some example then add a comment to explain what it is.